readthedocs/doc_builder/python_environments.py

```diff
@@ -128,8 +129,14 @@ def install_core_requirements(self):
         if self.project.documentation_type == 'mkdocs':
             requirements.append('mkdocs==0.15.0')
         else:
-            requirements.extend(['sphinx==1.5.3', 'sphinx-rtd-theme<0.3',
-                                 'readthedocs-sphinx-ext<0.6'])
+            if self.project.has_feature(Feature.USE_SPHINX_LATEST):
+                requirements.append('sphinx<1.7')
```

Comment:
If it's the latest, shouldn't it just do `sphinx` without a version ID?

Reply:
I was curious what we wanted to do here. I'm fine with leaving it bleeding edge, without a specifier. '1.7' specifier was a stopgap from making the version go too high.

Reply:
Maybe do semvar and `<2`?

Reply:
ah yeah, that's a really good idea. hopefully they follow semver as well. i'll do that instead of open ended on this and `setuptools` PR.